analyzers/rolaguard_base_analyzer/CheckDuplicatedSession.py

```diff
@@ -9,8 +9,12 @@
 class CheckDuplicatedSession():
     def __init__(self):
         self.last_packet = defaultdict(lambda: {})
+        self.last_gc = None
+
 
     def __call__(self, packet, device_session, device, gateway, policy):
+        if self.last_gc is None: self.last_gc = packet.date
+        if (self.last_gc - packet.date).seconds > 3600: self.garbage_collection(today = packet.date)
```

Comment:
**self.last_gc** will be lower than **packet.date**, and this may provoke some unexpected behaviour. I think it should be:
`if (packet.date - self.last_gc) ...`

Reply:
Yeah, I don't know how that worked locally. Done!